Describe the effect of disable_active_migration more accurately (#403)
quicwg/multipath: draft-ietf-quic-multipath.md

# Path Management {#path-management}

After completing the handshake, endpoints have agreed to enable
multipath support. They can also start using multiple paths when both endpoints
have issued available connection IDs for at least one unused Path ID.

> If endpoint receives a disable_active_migration transport parameter

Reviewer: ```suggestion
If an endpoint receives a disable_active_migration transport parameter
```

provided by the peer, it is forbidden to use a new local address

> to establish new paths for the peer address used during handshake,

Reviewer 2: ```suggestion
to establish new paths to the same peer address as used during handshake,
```

> to establish new paths to the same peer address as used during handshake,
> but establishment of additional paths to other peer addresses

Reviewer: ```suggestion
to establish new paths to the peer's handshake address. However,
establishment of additional paths from any local address to other peer addresses
```

I think we need to state that this TP also doesn't restrict the endpoint to a single local address.
